config.json

```diff
@@ -1060,6 +1060,18 @@
       "unlocked_by": "tree-building",
       "uuid": "7aa53a27-4ff3-4891-809f-2af728eb55a0"
     },
+    {
+      "core": false,
+      "difficulty": 4,
+      "slug": "markdown",
+      "topics": [
+        "refactoring",
+        "strings",
+        "text_formatting"
+      ],
+      "unlocked_by": "tree-building",
```

Comment:
In answer to your question about the "unlocked_by" property, it must be another exercise which has "core": true.  For this exercise, "tree-building" does qualify since it is a core exercise.  Generally the unlocked_by exercise would have some intersecting "topics" property. The one common property between tree-building and markdown is "refactoring".  There might be a better core exercise for topics intersection.  See what you think.  The "core" and "unlocked_by" are items in preparation for next revision of exercism user interface which is still in the works.